bedrock/thunderbird/redirects.py

```diff
@@ -1,58 +0,0 @@
-from bedrock.redirects.util import redirect
-
-
-redirectpatterns = (
-    # bug 1124038
-    redirect(r'^thunderbird/organizations/(?:all-esr\.html|faq/?)$', 'thunderbird.organizations'),
-
-    # bug 1124042
-    redirect(r'^thunderbird/features/email_providers\.html$', 'thunderbird.email-providers'),
```

Comment:
Should be redirected to https://www.thunderbird.net/en-US/email-providers/ as it turns out it is used in Thunderbird.

Reply:
Added 👍